src/client/components/Nav.jsx

```diff
@@ -39,80 +39,82 @@ class Nav extends React.Component {
+         ${determinePanelStyles('/military-service', '/military-service/additional-information')}`}>
           <div>
-            <h5>Insurance Information</h5>
+            <h5>Military Service</h5>
             <ul className="usa-unstyled-list">
-              <li className={`${determineSectionStyles('/insurance-information/general')}`}>
-                General Insurance
+              <li className={`${determineSectionStyles('/military-service/service-information')}`}>
+                Basic Information
               </li>
-              <li className={`${determineSectionStyles('/insurance-information/medicare-medicaid')}`}>
-                Medicare/Medicaid
+              <li className={`${determineSectionStyles('/military-service/additional-information')}`}>
+                Service History
               </li>
             </ul>
           </div>
         </li>
         <li role="presentation" className={`three ${subnavStyles}
-         ${determinePanelStyles('/military-service', '/military-service/additional-information')}`}>
+         ${determinePanelStyles('/va-benefits', '/va-benefits/basic-information')}`}>
           <div>
-            <h5>Military Service</h5>
+            <h5>VA Benefits</h5>
             <ul className="usa-unstyled-list">
-              <li className={`${determineSectionStyles('/military-service/service-information')}`}>
-                Service
-              </li>
-              <li className={`${determineSectionStyles('/military-service/additional-information')}`}>
-                Additional Military
+              <li className={`${determineSectionStyles('/va-benefits/basic-information')}`}>
+                Basic Information
               </li>
             </ul>
           </div>
         </li>
         <li role="presentation" className={`four ${subnavStyles}
-         ${determinePanelStyles('/financial-assessment', '/financial-assessment/deductible-expenses')}`}>
+         ${determinePanelStyles('/household-information', '/household-information/deductible-expenses')}`}>
           <div>
-            <h5>Financial Assessment</h5>
+            <h5>Household Information</h5>
             <ul className="usa-unstyled-list">
-              <li className={`${determineSectionStyles('/financial-assessment/financial-disclosure')}`}>
+              <li className={`${determineSectionStyles('/household-information/financial-disclosure')}`}>
                 Financial Disclosure
               </li>
-              <li className={`${determineSectionStyles('/financial-assessment/spouse-information')}`}>
+              <li className={`${determineSectionStyles('/household-information/spouse-information')}`}>
                 Spouse
               </li>
-              <li className={`${determineSectionStyles('/financial-assessment/child-information')}`}>
+              <li className={`${determineSectionStyles('/household-information/child-information')}`}>
                 Child
```

Comment:
I know the original form called this the `Child` section, but I wonder if it makes more sense to call it the `Children` section?